test/geometry/utility/geometry_test.cc

```diff
@@ -177,6 +178,218 @@ TEST_P(ComputeCenterlineTest, Test) {
 INSTANTIATE_TEST_CASE_P(ComputeCenterlineTestGroup, ComputeCenterlineTest,
                         ::testing::ValuesIn(LeftRightCenterlineTestCases()));
 
+struct LineStringPointAtPCase {
+  LineString3d line_string{};
+  double p{};
+  Vector3 expected_point{};
+};
+
+std::vector<LineStringPointAtPCase> LineStringPointAtPTestCases() {
+  return {
+      {
+          LineString3d{{0., 0., 0.}, {10., 0., 0.}} /* line string*/,
+          double{5.} /* p */,
```

Comment:
Do you need the double? I think it is taken by default as double if you use the `.`.

Reply:
Correct. Done.